vlib/v/fmt/tests/infix_expr_expected.vv

```diff
@@ -24,8 +24,7 @@ fn main() {
 			{
 				// Indent this much to force a break after the assign (`:=`).
 				// Check that the trailing space is removed
-				should_cast :=
-					(g.table.type_kind(stmt.left_types.first()) in js.shallow_equatables)
+				should_cast := g.table.type_kind(stmt.left_types.first()) in js.shallow_equatables
 					&& (g.cast_stack.len <= 0 || stmt.left_types.first() != g.cast_stack.last())
```

Comment:
this comment is no longer valid; imho add one more nested {}, so that there is a break after `:=` again

Reply:
Ok, I'll change it.